packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -339,49 +322,48 @@ const Select = <
         event.stopPropagation();
 
       return (
-        <ChipsInnerContainer
-          isInteractive={isInteractive}
-          odysseyDesignTokens={odysseyDesignTokens}
-        >
-          {selection.length <= 0 && (
-            <ChipsSpacer odysseyDesignTokens={odysseyDesignTokens} />
-          )}
-          {selection.map(
-            (item: string) =>
-              item?.length > 0 && (
-                <MuiChip
-                  key={item}
-                  label={
-                    <>
-                      {item}
-                      {!isInteractive &&
-                        controlledStateRef.current === CONTROLLED &&
-                        hasMultipleChoices && (
-                          <NonInteractiveIcon
-                            odysseyDesignTokens={odysseyDesignTokens}
-                          />
-                        )}
-                    </>
-                  }
-                  tabIndex={-1}
-                  onDelete={
-                    isInteractive && controlledStateRef.current === CONTROLLED
-                      ? () => removeSelection(item)
-                      : undefined
-                  }
-                  deleteIcon={
-                    <CloseCircleFilledIcon
-                      sx={{ pointerEvents: "auto" }}
-                      // We need to stop event propagation on mouse down to prevent the deletion
-                      // from being blocked by the Select list opening, and also ensure that
-                      // the pointerEvent is registered even when the parent's are not
-                      onMouseDown={stopPropagation}
-                    />
-                  }
-                />
-              ),
-          )}
-        </ChipsInnerContainer>
+        Array.isArray(internalSelectedValues) && (
```

Comment:
@jordankoschei-okta Will this ever NOT be an array?

Reply:
Yes! If it's not a multi-select it won't be an array; in that case, it needs to return `undefined` so that the `renderValue` prop on MuiSelect is also undefined and reverts to the default way of rendering values.

Reply:
We only ever render `Chips` if `hasMultipleChoices` is true right? So, wouldn't this always be an array in that case?

Reply:
Actually, we already check if the value is an array [here](https://github.com/okta/odyssey/pull/2183/files/bfbb8e086a5542650041e40fadc68ecd3c6c23a2#diff-1e4d080b7bbf21c0a69512d075fad69a8ea134c102a54faefc5daae234a57eeaR415) so, this extra check seems unnecessary

Reply:
Unless I am missing something. Which, is totally possible